pkg/kritis/apis/kritis/v1beta1/register.go

```diff
@@ -47,6 +47,7 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(SchemeGroupVersion,
 		&ImageSecurityPolicy{},
 		&ImageSecurityPolicyList{},
+		&AttestationAuthority{},
```

Comment:
Shd we not add AttestationAuthorityList here as well?

Reply:
done